From 2a31fc114dae3ddb23a816428f00c154dac5ee95 Mon Sep 17 00:00:00 2001
From: Guenter Roeck <linux@roeck-us.net>
Date: Sun, 21 Apr 2013 09:13:28 -0700
Subject: [PATCH 18/19] hwmon: (nct6775) Fix coding style problems

Add space around binary operators (CodingStyle, chapter 3.1).

Signed-off-by: Guenter Roeck <linux@roeck-us.net>
(cherry picked from commit 6445e6600fa632448cac64e71119310378464ad9)
---
 drivers/hwmon/nct6775.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/hwmon/nct6775.c
+++ b/drivers/hwmon/nct6775.c
@@ -833,7 +833,7 @@ static void nct6775_update_fan_div(struc
 	data->fan_div[1] = (i & 0x70) >> 4;
 	i = nct6775_read_value(data, NCT6775_REG_FANDIV2);
 	data->fan_div[2] = i & 0x7;
-	if (data->has_fan & (1<<3))
+	if (data->has_fan & (1 << 3))
 		data->fan_div[3] = (i & 0x70) >> 4;
 }
 
@@ -1076,7 +1076,7 @@ static struct nct6775_data *nct6775_upda
 
 	mutex_lock(&data->update_lock);
 
-	if (time_after(jiffies, data->last_updated + HZ + HZ/2)
+	if (time_after(jiffies, data->last_updated + HZ + HZ / 2)
 	    || !data->valid) {
 		/* Fan clock dividers */
 		nct6775_update_fan_div_common(data);
@@ -1177,7 +1177,7 @@ store_in_reg(struct device *dev, struct
 		return err;
 	mutex_lock(&data->update_lock);
 	data->in[nr][index] = in_to_reg(val, nr);
-	nct6775_write_value(data, data->REG_IN_MINMAX[index-1][nr],
+	nct6775_write_value(data, data->REG_IN_MINMAX[index - 1][nr],
 			    data->in[nr][index]);
 	mutex_unlock(&data->update_lock);
 	return count;
